README.md

```diff
@@ -1,2 +1,136 @@
 # word2gauss
 Gaussian word embeddings
+
+Python/Cython implementation of [Luke Vilnis and Andrew McCallum
+<i>Word Representations via Gaussian Embedding</i>, ICLR 2015](http://arxiv.org/abs/1412.6623)
+that represents each word as a multivariate Gaussian.
+Scales to (relatively) large corpora using Cython extensions and threading
+with asynchronous stochastic gradient descent (Adagrad).
+
+## Getting started
+
+### Installing
+First install numpy, scipy and the packages in `requirements.txt`.
+Then `make install`.  `make test` runs the test suite.
+
+### Code overview
+The `GaussianEmbedding` class is the main workhorse for most tasks.  It
+stores the model data, deals with serialization to/from files and
+learns the parameters.  To allow embedding of non-word types like
+hierarchies and entailment relations, `GaussianEmbedding` has
+no knowledge of any vocabulary and operates only on
+unique IDs.  Each ID is a `uint32` from `0 .. N-1` with `-1` signifying
+an OOV token.
```

Comment:
I'm sure OOV may be a well-known acronym inside the NLP community, but I would probably expand this out at least on its first usage. I'm assuming this means "out-of-vocabulary".